i6_models/parts/conformer/feedforward.py

```diff
@@ -0,0 +1,45 @@
+from torch import nn
+
+
+class ConformerFeedForwardv1(nn.Module):
+    """
+    Conformer feedforward module
+    """
+
+    def __init__(self, input_dim, hidden_dim, dropout=0.1):
+        """
+        :param int input_dim: input dimension
+        :param int hidden_dim: hidden dimension (normally set to 4*input_dim as suggested by the paper)
+        :param float dropout: dropout probability
+        """
+        super().__init__()
+
+        self.layer_norm = nn.LayerNorm(normalized_shape=input_dim)
```

Comment:
In our earlier implementation in RETURNN-common, we decided to have layer-norm not as part of this module, but rather as part of the outer `ConformerEncoderLayer`.